code-challenges.js

```diff
@@ -31,38 +36,122 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const boilingPoint = (water) => {
+if (water > 212) return "yes"
+else if (water < 212) return "no"
+else return ("perfect")
+}
+
+// console.log(boilingPoint(temp1))
+// console.log(boilingPoint(temp2))
+// console.log(boilingPoint(temp3))
+
+
+
+//output yes, no, perfect.
+
+
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: first combine the 2 arrays using concat.
+// then create a const that is the value of the 2 arrays combined
+//then run the new array in console log with .length 
+
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+// const CombinedArray = padres1998WorldSeriesRuns.concat(padres1984WorldSeriesRuns)
+
+// console.log(CombinedArray.length)
+
+
+//output 9
 
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: make string into array using .split
+// reverse aray using .reverse
+// join array and make it a string using .join
+
 
 const currentCohort = "Hotel 2022"
 // Expected output: "2202 letoH"
 
+
+// const splitString = currentCohort.split("")
+
+// const reverseArray = splitString.reverse()
+
+// const newString = reverseArray.join("")
+
+// console.log(newString) 
+
+
+//output: 2202 letoH
+
+
+//i feel as if there is another way to do this but im not sure how other than this way
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Pseudo code:
 
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
 // Expected output: 7
-
 const givenValue2 = 10
+
+// {
+//  myNumbers.lastIndexOf(givenValue1)
```

Comment:
So this section is actually not doing anything, The brackets aren't connecting this to a function but you got the console.logs! I wanted to point this out but you got the correct implementation in the console.log's so really good job!